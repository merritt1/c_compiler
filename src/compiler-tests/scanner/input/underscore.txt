identifier
_identifier
_identifier_last

