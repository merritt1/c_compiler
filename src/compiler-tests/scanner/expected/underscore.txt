loc = 0001:0001-0001:0010     text = identifier               token = [IDENTIFIER          ]     name = identifier
loc = 0002:0001-0002:0011     text = _identifier              token = [IDENTIFIER          ]     name = _identifier
loc = 0003:0001-0003:0016     text = _identifier_last         token = [IDENTIFIER          ]     name = _identifier_last
